applications/crossbar/src/modules/cb_ledgers.erl

```diff
@@ -152,6 +159,23 @@ validate(Context, ?CREDIT) ->
     cb_context:validate_request_data(<<"ledgers">>, Context);
 validate(Context, ?DEBIT) ->
     cb_context:validate_request_data(<<"ledgers">>, Context);
+validate(Context, ?TOTAL) ->
+    case kz_ledgers:total_sources(cb_context:account_id(Context)) of
+        {'error', _R} ->
+            lager:debug("failed to fetch ledgers total: ~p", [_R]),
+            cb_context:add_system_error('unspecified_fault', Context);
```

Comment:
Why not passing `_R` here. it already have crossbar errors.

Reply:
@icehess because we can't guaranty the type of _R and should be a rare occurrence

Reply:
@icehess Because the type of _R is not guaranteed and should be a rare occurrence